descriptors/ModuleDescriptor-template.json

```diff
@@ -101,7 +101,8 @@
             "inventory-storage.electronic-access-relationships.collection.get",
             "configuration.entries.collection.get",
             "inventory-storage.holdings-note-types.collection.get",
-            "inventory-storage.item-note-types.collection.get"
+            "inventory-storage.item-note-types.collection.get",
+            "inventory-storage.call-number-types.collection.get"
```

Comment:
Isn't call-number-type providing a separate interface, like the other reference data?
Please verify. We need to add interface dependency as well

Reply:
@KVupp oops, you are absolutely right! I will push changes

Reply:
oops, you absolutely right! Will push changes in 15 min